packages/openapi3/src/lib.ts

```diff
@@ -104,6 +115,16 @@ const EmitterOptionsSchema: JSONSchemaType<OpenAPI3EmitterOptions> = {
         "  - `openapi.Org1.Service2.v1.1.yaml`    ",
       ].join("\n"),
     },
+    "openapi-versions": {
+      type: "array",
+      items: {
+        type: "string",
+        enum: ["3.0.0", "3.1.0"],
+        nullable: true,
+        description: "The versions of OpenAPI to emit. Defaults to `v3.0`",
```

Comment:
`v3.0` isn't exactly the string that this defaults to. Suggest: "Default: `["3.0.0"]`"

Reply:
Fixed!